.coveragerc

```diff
@@ -394,6 +394,7 @@ omit =
     homeassistant/components/freebox/switch.py
     homeassistant/components/fritz/common.py
     homeassistant/components/fritz/device_tracker.py
+    homeassistant/components/fritz/image.py
```

Comment:
even I've added some testes it is on exception list, beacuse I did not get it managed to test the `async_image` itself